R/annotation.R

```diff
@@ -50,6 +50,13 @@ annotate_ssm_blacklist = function(mutations_df,
   blacklist_list = list()
   for(b in blacklist_files){
     full_path = paste0(project_base,b)
+
+    #check for missingness
+    if(!file.exists(full_path)){
+      message("Cannot find file locally. If working remotely, perhaps you forgot to load your config (see below) or sync your files?")
```

Comment:
Can you also print out the full path before this message i.e. print(full_path) so the user knows what file is missing?

Reply:
Absolutely!